core/esmf-aspect-model-aas-generator/src/main/java/org/eclipse/esmf/aspectmodel/aspect/AASModelAspectGenerator.java

```diff
@@ -0,0 +1,57 @@
+package org.eclipse.esmf.aspectmodel.aspect;
+
+import org.apache.jena.rdf.model.Model;
+import org.eclipse.digitaltwin.aas4j.v3.model.Environment;
+import org.eclipse.esmf.aspectmodel.resolver.services.SammAspectMetaModelResourceResolver;
+import org.eclipse.esmf.aspectmodel.resolver.services.VersionedModel;
+import org.eclipse.esmf.aspectmodel.serializer.PrettyPrinter;
+import org.eclipse.esmf.aspectmodel.serializer.RdfModelCreatorVisitor;
+import org.eclipse.esmf.aspectmodel.vocabulary.Namespace;
+import org.eclipse.esmf.metamodel.Aspect;
+
+import java.io.*;
+import java.util.function.Function;
+
+public class AASModelAspectGenerator {
```

Comment:
I think there should be only one class, not both `AASModelAspectGenerator` and `AASModelAspectTranslator`. Please consolidate them into one class. In addition to a constructor that takes the `Environment` as input, you might want to add a static convenience function where you pass in an InputStream.

Reply:
You know, I think we can talk...

But I think this is two different options. Because in `AASModelAspectGenerator ` we just generate translated Model  of Aspect to .ttl file. In `AASModelAspectTranslator ` we translate `Environment ` to the real Aspect. Also, I follow our structure for Visitors.